zppy/templates/e3sm_diags.bash

```diff
@@ -88,6 +88,17 @@ done
 cd ..
 {%- endif %}
 
+{%- if "streamflow" in sets %}
+# TODO: generalize "native"
+rofDir="{{ output }}/post/rof/native/ts/monthly/{{ ts_num_years }}yr"
```

Comment:
Is it ok to leave `native` and `monthly` hard-coded or do we need to consider other options?